Include/internal/pycore_frame.h

```diff
@@ -76,7 +76,12 @@ typedef struct _PyInterpreterFrame {
     _PyStackRef *stackpointer;
     uint16_t return_offset;  /* Only relevant during a function call */
     char owner;
+#ifdef Py_DEBUG
+    char visited:4;
+    char lltrace:4;
```

Comment:
Why 4?
You only need one bit (although it is best to use `uint8_t` as the signedness of `char` is implementation defined.